modules/storage-ephemeral-vols-overview.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * storage/generic-ephemeral-vols.adoc
+
+:_content-type: CONCEPT
+[id="generic-ephemeral-vols-overview_{context}"]
+= Overview
+
+Generic ephemeral volumes are a type of ephemeral volume that can be provided by all storage drivers that also support persistent volumes and dynamic provisioning. Generic ephemeral volumes are similar to `emptyDir` volumes in that they provide a per-pod directory for scratch data, which is usually empty after provisioning. Generic ephemeral volumes are specified inline in the pod spec.
```

Comment:
You might want to mention in here that "Generic ephemeral volumes follow the Pod's lifecycle and get created and deleted along with the Pod." This is a slightly different model from a PV, which can exist without being attached to a pod.